modules/ding_nodelist/ding_nodelist.module

```diff
@@ -1488,6 +1488,48 @@ function _ding_nodelist_format_event_time(array $date) {
   return $result;
 }
 
+/**
+ * Gets the location off a event if defined else the library.
+ *
+ * @param mixed $item
+ *   The event item.
+ * @param bool $islink
+ *   Return formatted as link.
+ *
+ * @return string
```

Comment:
return render array

Reply:
There is Scrutinizer error message. But is says that Drupal code guidelines are not compatible with the current version.